mne/io/hitachi/hitachi.py

```diff
@@ -0,0 +1,278 @@
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import datetime as dt
+import re
+
+import numpy as np
+
+from ..base import BaseRaw
+from ..constants import FIFF
+from ..meas_info import create_info
+from ..nirx.nirx import _read_csv_rows_cols
+from ..utils import _mult_cal_one
+from ...utils import (logger, verbose, fill_doc, warn, _check_fname,
+                      _check_option)
+
+
+@fill_doc
+def read_raw_hitachi(fname, preload=False, verbose=None):
+    """Reader for a Hitachi fNIRS recording.
+
+    Parameters
+    ----------
+    fname : str
+        Path to the Hitachi CSV file.
+    %(preload)s
+    %(verbose)s
+
+    Returns
+    -------
+    raw : instance of RawHitachi
+        A Raw object containing Hitachi data.
+
+    See Also
+    --------
+    mne.io.Raw : Documentation of attribute and methods.
+
+    Notes
+    -----
+    %(hitachi_notes)s
+    """
+    return RawHitachi(fname, preload, verbose=verbose)
+
+
+def _check_bad(cond, msg):
+    if cond:
+        raise RuntimeError(f'Could not parse file: {msg}')
+
+
+@fill_doc
+class RawHitachi(BaseRaw):
+    """Raw object from a Hitachi fNIRS file.
+
+    Parameters
+    ----------
+    fname : str
+        Path to the Hitachi CSV file.
+    %(preload)s
+    %(verbose)s
+
+    See Also
+    --------
+    mne.io.Raw : Documentation of attribute and methods.
+
+    Notes
+    -----
+    %(hitachi_notes)s
+    """
+
+    @verbose
+    def __init__(self, fname, preload=False, *, verbose=None):
+        fname = _check_fname(fname, 'read', True, 'fname')
+        logger.info('Loading %s' % fname)
+
+        raw_extra = dict(fname=fname)
+        info_extra = dict()
+        subject_info = dict()
+        ch_wavelengths = dict()
+        fnirs_wavelengths = [None, None]
+        meas_date = age = ch_names = sfreq = None
+        with open(fname, 'rb') as fid:
+            lines = fid.read()
+        lines = lines.decode('latin-1').rstrip('\r')
+        oldlen = len(lines)
+        assert len(lines) == oldlen
+        bounds = [0]
+        bounds.extend(a.end() for a in re.finditer('\r', lines))
+        bounds.append(len(lines))
+        lines = lines.split('\r')
+        assert len(bounds) == len(lines) + 1
+        line = lines[0].rstrip(',')
+        _check_bad(line != 'Header', 'no header found')
+        li = 0
+        mode = None
+        for li, line in enumerate(lines[1:], 1):
+            # Newer format has some blank lines
+            if len(line) == 0:
+                continue
+            parts = line.rstrip(',').split(',')
+            if len(parts) == 0:  # some header lines are blank
+                continue
+            kind, parts = parts[0], parts[1:]
```

Comment:
I needed to make this change for the code to work on my computer. Could this be an OS dependent thing? If relevant I am on a Mac.

```suggestion
            kind, parts = parts[0], parts[1:]
            kind = kind.replace("\n", "")
```